ARCHITECTURE.md

```diff
@@ -114,9 +114,9 @@ Well, we can fix this. The price is hard-to-read type errors if you miss somethi
 So we can simply use `Rank2Types` extension and use a layer that has the constraints defined *per function*:
 ```
 data LoggingLayer = LoggingLayer
-    { iolLogDebug   :: forall m. (MonadIO m) => Text -> m ()
-    , iolLogInfo    :: forall m. (MonadIO m) => Text -> m ()
-    , iolNonIo      :: forall m. (MonadThrow m) => m ()
+    { logDebug   :: forall m. (MonadIO m) => Text -> m ()
```

Comment:
It's very usual to have a prefix on the field names so we avoid namespace collision.
So maybe it would be better to call them `llLogDebug`?